src/Wpf/Prism.Wpf/PrismInitializationExtensions.cs

```diff
@@ -66,7 +66,7 @@ internal static void RegisterDefaultRegionAdapterMappings(this RegionAdapterMapp
         internal static void RunModuleManager(IContainerProvider containerProvider)
         {
             IModuleManager manager = containerProvider.Resolve<IModuleManager>();
-            manager.Run();
+            manager?.Run();
```

Comment:
If the manager was null you would have gotten a ContainerResolutionException when resolving IModuleManager on line 68